relayer/relayer.go

```diff
@@ -66,21 +66,20 @@ func (r *Relayer) route(msgs []*message.Message) {
 	}
 
 	for _, m := range msgs {
+		log.Debug().Msgf("Routing message %+v to destination %v", m.String(), destChain.DomainID())
 		r.metrics.TrackDepositMessage(m)
-
 		for _, mp := range r.messageProcessors {
 			if err := mp(m); err != nil {
-				log.Error().Err(fmt.Errorf("error %w processing mesage %v", err, m))
+				log.Error().Err(fmt.Errorf("error %w processing mesage %v", err, m.String()))
```

Comment:
This might look weird in the logs.
Would it make more sense to add just some message ID here and paste the data as an object associated in the log